packages/ses/docs/lockdown.md

```diff
@@ -366,12 +367,54 @@ the container to exit explicitly, and we highly recommend setting
 - `'none'`: do not install traps for uncaught exceptions. Errors are likely to
   appear as `{}` when they are reported by the default trap.
 
+## `evalTaming` Options
+
+This option only affects the initial compartment!
+
+**Background**: For every compartment including the implied compartment in the
+initial JavaScript realm, there are evaluators `eval` and `Function`.
```

Comment:
Whether the realm is the initial realm doesn't matter. The description here applies to the compartments of any realm.

```suggestion
**Background**: Every realm has an implicit initial compartment we call the "start compartment". Explicit compartments are made with the `Compartment` constructor.
For every compartment including the start compartment, there are evaluators `eval` and `Function`.
```

This does seem like a weird place to define "start compartment" and "explicit compartment". If they are adequately explained elsewhere, we can delete my first sentence above.